packages/rpc/src/__tests__/client.test.ts

```
@@ -0,0 +1,472 @@
+			mockFetch.mockResolvedValue(mockResponse);
+
+			const client = createClient<typeof testRouter>({
+				baseUrl: "https://api.example.com",
+			});
+
+			// The error handling is in the jfetch function
+			expect(client).toBeDefined();
+		});
+
+		it("should serialize query parameters with SuperJSON", () => {
+			const client = createClient<typeof testRouter>({
```

Biome (1.9.4)

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend client with an underscore.


(lint/correctness/noUnusedVariables)